.gitignore

```diff
@@ -0,0 +1 @@
+credentials.env
```

Comment:
We should continue using `.env` as the file name since that’s as close to a standard as I’ve seen.

Reply:
For what it's worth, I ran a Twitter poll and got 5 responses: https://twitter.com/benzado/status/1213116276371075078